test/library/packages/Crypto.skipif

```diff
@@ -12,14 +12,14 @@ Ok = False
 if platform != 'linux64':
     Ok = False  # not linux
 else:
-
-    v = subprocess.check_output(['pkg-config', '--modversion', 'libcrypto'])
-    v = v.rstrip()
-    v = str(v, encoding='utf-8', errors='surrogateescape')
-
-    if parse_version(v) >= parse_version('1.1'):
-        # OK, version is compatible.
-        Ok = True
-
+    sub = subprocess.run(['pkg-config', '--modversion', 'libcrypto'],
+                         capture_output=True)
```

Comment:
Based on the code here, you probably saw that `--modversion` fails if the package isn't installed and the `parse_version` isn't very good for pkg-config output. `pkg-config` actually has built-in version comparisons, so I think we improve the portability of this and simplify it with:

```python
    sub = subprocess.run(['pkg-config', '--exists', 'libcrypto >= 1.1'])
    if sub.returncode == 0:
        Ok = True
```

Reply:
And just some sample output on a machine without libcrypto and with it:

Without:
```sh
chaplogin01:~> pkg-config --modversion libcrypto
No package 'libcrypto' found
chaplogin01:~> pkg-config --exists 'libcrypto >= 1.1'
chaplogin01:~> echo $?
1
```

With:
```sh
chapcs11: ~ $ pkg-config --modversion libcrypto
1.1.1k
chapcs11: ~ $ pkg-config --exists 'libcrypto >= 1.1'
chapcs11: ~ $ echo $?
0
```

With, but newer version:

```sh
chapcs11: ~ $ pkg-config --exists 'libcrypto >= 1.2'
chapcs11: ~ $ echo $?
1
```


```

Reply:
Thanks for the suggestion. I think it's a great idea & I've made the change.